content/components/autocomplete.mdx

```diff
@@ -6,83 +6,143 @@ status: Alpha
+  />
+  <Box as="p" mt="0">
+    If no options are available based on the search term, the menu will display a message that says "No selectable
+    options". A concise, custom message may be used in place of the default.
+  </Box>
 </Box>
 
-<Box mb={3} display="flex" alignItems="flex-start" flexDirection={["column", "column", "column", "column", "row"]} sx={{gap: 4}}>
-    <img
-        width="456"
-        alt="menu opened, showing an empty state"
-        src="https://user-images.githubusercontent.com/2313998/138128811-5542f1e0-2177-4741-af5b-c4bf52c65401.png"
-    />
-    <Box as="p" mt="0">A loading indicator should be displayed while the data for the list of options is being populated. The loading indicator helps the user understand that the list is not empty, it's just not done loading.</Box>
+<Box
+  mb={3}
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 4}}
+>
+  <img
+    width="456"
+    alt="opened autocomplete menu with an add item action"
+    src="https://user-images.githubusercontent.com/18661030/170589959-73608127-771d-4f7e-a78d-11b31433141d.png"
+  />
+  <Box as="p" mt="0">
+    If a user is not limited to a pre-defined list of options, an additional item can be rendered in the menu to select
+    the value that has been typed into the text input.
+  </Box>
 </Box>
 
-#### Menu item rendering
-
-By default, menu items are rendered as a single line of text. The list in the menu is rendered using the [Action List](/action-list) component, so menu items can be rendered with all of the same options as Action List items.
+<Box
+  mb={3}
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 4}}
+>
+  <img
+    width="456"
+    alt="opened autocomplete menu with a loading spinner"
+    src="https://user-images.githubusercontent.com/18661030/170589963-ab3e211b-1082-4fdb-928e-eddd2b773ba3.png"
+  />
+  <Box as="p" mt="0">
+    A loading indicator should be displayed while the data for the list of options is being populated. The loading
+    indicator helps the user understand that the list is not empty, it's just not done loading.
+  </Box>
+</Box>
 
 ## Sort and filter behavior
 
 ### Sorting
 
-The order of the items should not be random: they should be ordered in a way that makes it easy for a user to find a specific value. This could mean items may be ranked by how likely a user is to pick that option (for example, ordering labels by the number of times they've been used in that repository), or it could simply be in alphabetical order.
+The order of the items should not be random– they should be ordered in a way that makes it easy for a user to find a specific value. This could mean items may be ranked by how likely a user is to pick that option (for example, ordering labels by the number of times they've been used in that repository), or it could simply be in alphabetical order.
```

Comment:
```suggestion
The order of the items should be ordered in a way that makes it easy for a user to find a specific value. This could mean items may be ranked by how likely a user is to pick that option (for example, ordering labels by the number of times they've been used in that repository), or it could simply be in alphabetical order.
```